docs/README-agent-install.md

```diff
@@ -1,82 +1,83 @@
-* The Cisco IOS XR Puppet implementation requires open source Puppet version 4.3.2 or Puppet Enterprise 2015.3.2
-* The following table lists supported platforms and OS versions.
+Puppet Versions |
+:--|
+Open Source Puppet 4.0 |
+Puppet Enterprise 2015.2 |
+<br>
 
-Platform           | OS     | OS Version           |
--------------------|--------|----------------------|
-Cisco Nexus N9k    | NX-OS  | 7.0(3)I2(1) and later
+Supported Platforms | OS    | OS Version           |
+:-------------------|-------|----------------------|
+Cisco Nexus N9k <br>*[How to get a virtual Nexus N9k](#VIRT_9K)* | NX-OS  | 7.0(3)I2(1) and later
 Cisco Nexus N3k    | NX-OS  | 7.0(3)I2(1) and later
 Cisco Nexus N5k    | NX-OS  | 7.3(0)N1(1) and later
 Cisco Nexus N6k    | NX-OS  | 7.3(0)N1(1) and later
 Cisco Nexus N7k    | NX-OS  | 7.3(0)D1(1) and later
 Cisco Nexus N8k    | NX-OS  | 7.0(3)F1(1) and later
+<br>
 
-Please note: A virtual Nexus N9000/N3000 may be helpful for development and testing. Users with a valid [cisco.com](http://cisco.com) user ID can obtain a copy of a virtual Nexus N9000/N3000 by sending their [cisco.com](http://cisco.com) user ID in an email to <get-n9kv@cisco.com>. If you do not have a [cisco.com](http://cisco.com) user ID please register for one at [https://tools.cisco.com/IDREG/guestRegistration](https://tools.cisco.com/IDREG/guestRegistration)
 
-#### Disk space
+Resource| Recommended| |
+:--|:--:|:--|
+Disk   | **400 MB** | Minimum free space before installing Puppet agent |
 
-400MB of free disk space on bootflash is recommended before installing the
-Puppet agent software.
+<br>
 
-#### Environment
+#### *Step 2. Choose an environment for running Puppet Agent*
 
-NX-OS supports three possible environments for running third party software:
-`bash-shell`, `guestshell` and the `open agent container (OAC)`.
+NX-OS Environment | Supported Platforms | |
+:--|:--:|:--|
+`bash-shell` | N3k, N8k, N9k | This is the native WRL Linux environment underlying NX-OS. It is disabled by default on NX-OS. |
+`guestshell` | N3k, N8k, N9k | This is a secure Linux container environment running CentOS. It is enabled by default in most platforms that support it. |
+`open agent`<br>`container (OAC)` | N5k, N6k, N7k | This is a 32-bit CentOS-based container created specifically for running Puppet Agent software. |
 
-Environment                  | Supported Platforms                      |
------------------------------|------------------------------------------|
-`bash-shell`                 | Cisco Nexus N3k, N8k, N9k                |
-`guestshell`                 | Cisco Nexus N3k, N8k, N9k                |
-`open agent container (OAC)` | Cisco Nexus N5k, N6k, N7k                |
+* *OAC containers are created for specific platforms and must be downloaded from [cisco.com](http://cisco.com). The OAC must be installed before the Puppet Agent can be installed.*
```

Comment:
Instead of linking to cisco.com here, suggest you just link directly to the download info in the doc.

https://github.com/chrisvanheuveln/cisco-network-puppet-module/blob/ff2a1aa08b155e5c29f3f325b4db02c9aba4aeba/docs/README-agent-install.md#step-1-download-the-oac-ova-file